src/qt/overviewpage.cpp

```diff
@@ -289,6 +273,13 @@ void OverviewPage::setWalletModel(WalletModel *model)
             privateSendStatus(true);
         });
 
+        // Disable privateSendClient builtin support for automatic backups while we are in GUI,
+        // we'll handle automatic backups and user warnings in privateSendStatus()
+        walletModel->privateSend().disableAutobackups();
+
+        timer = new QTimer(this);
+        connect(timer, SIGNAL(timeout()), this, SLOT(privateSendStatus()));
```

Comment:
This breaks UI updates during mixing for me. Moving these 2 lines back to `OverviewPage` 
 ctor fixes the issue.

Reply:
Fixed and squashed already.. wonder why i even changed this :D